cargo-espflash/src/main.rs

```diff
@@ -545,7 +545,7 @@ fn build(
                     continue;
                 };
 
-                if package.name != "esp-idf-sys" {
+                if package.name.to_string() != "esp-idf-sys" {
```

Comment:
```suggestion
                if package.name.as_ref() != "esp-idf-sys" {
```

No need to clone the string to compare it